app/models/goal.py

```diff
@@ -13,7 +13,7 @@ def to_dict(self):
         return {
             "id": self.id,
             "title": self.title,
-            "tasks": [task.to_dict() for task in self.tasks]
+            # "tasks": [task.to_dict() for task in self.tasks]
```

Comment:
It would be preferable to _remove_ the unnecessary line rather than commenting it out.

In professional code leaving commented code invites questions about why the code is commented, and whether it should be reinstated every time someone comes across it. Keep your code clean.